### PR TITLE
Delete kid frontend

### PR DESCRIPTION
**Pull request by Cajan19**
Add delete button with yes/no confirm popup.

### CODE REVIEW

File: frontend-nannyway/src/components/accordion/ChildInCareAccordion.js [hunk 1]
```diff
@@ -173,7 +202,7 @@ export default function ChildInCareAccordion({kid}) {
                             <Divider/>
                             <List>
                                 <ListItem>
-                                {/*<ListItem button>*/}
+                                    {/*<ListItem button>*/}
```
Comment (fabianschmauder) [inline frontend-nannyway/src/components/accordion/ChildInCareAccordion.js:205]: remove comment

File: frontend-nannyway/src/components/accordion/ChildInCareAccordion.js [hunk 2]
```diff
@@ -57,6 +76,16 @@ export default function ChildInCareAccordion({kid}) {
...
+        setOpen(true);
+    };
+
+    const handleClose = () => {
+        setOpen(false);
+    };
+
     return (
         <div className={classes.root}>
             <Accordion expanded={expanded === 'panel1'} onChange={handleChange('panel1')}
```
Comment (fabianschmauder) [inline frontend-nannyway/src/components/accordion/ChildInCareAccordion.js:91]: why you dont use just true and false? do you plan to have more than one Accordion parts?

File: frontend-nannyway/src/components/accordion/ChildInCareAccordion.js [hunk 3]
```diff
@@ -182,6 +211,49 @@ export default function ChildInCareAccordion({kid}) {
...
+                                    <Button
+                                        variant="contained"
+                                        className={classes.deleteButton}
+                                        onClick={handleClickOpen}
+                                        startIcon={<DeleteForeverIcon/>}
+                                    >
+                                        {kid.firstName} löschen
+                                    </Button>
+                                </Box>
+                                <Dialog
```
Comment (fabianschmauder) [inline frontend-nannyway/src/components/accordion/ChildInCareAccordion.js:231]: you could move the dialog to a separate component like "ConfirmDialog"

File: frontend-nannyway/src/components/accordion/ChildInCareAccordion.js (hunk 2 above)
Comment (Cajan19) [reply frontend-nannyway/src/components/accordion/ChildInCareAccordion.js:91]: what? sry, i don´t get your point. please write loud and slowly 😆

**Comment by ThanhDuong2503** [comment]
best branch name ever! 👍🏻

File: frontend-nannyway/src/components/accordion/ChildInCareAccordion.js (hunk 3 above)
Comment (Cajan19) [reply frontend-nannyway/src/components/accordion/ChildInCareAccordion.js:223]: Ran into some issues regarding the useState. Will leave the code as it is for now, but opened an according todo for the next days ;)

File: frontend-nannyway/src/components/accordion/ChildInCareAccordion.js (hunk 2 above)
Comment (Cajan19) [reply frontend-nannyway/src/components/accordion/ChildInCareAccordion.js]: Done

File: frontend-nannyway/src/components/accordion/ChildInCareAccordion.js (hunk 1 above)
Comment (Cajan19) [reply frontend-nannyway/src/components/accordion/ChildInCareAccordion.js]: Done

**Pull request merged**